pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -276,6 +276,27 @@ static int is_simple_frame(PyObject *obj) {
     Py_DECREF(mgr);
     return ret;
 }
+/* TODO: Consider unifying with checknull and co.
+   in missing.pyx */
+static int is_null_obj(PyObject* obj) {
```

Comment:
Nit but I think this should be called `is_json_null` since its semantics may vary from what we have elsewhere in the codebase